src/local-editor-service.ts

```diff
@@ -140,6 +139,24 @@ export class LocalEditorService extends EditorService {
     };
   }
 
+  _generateAutoCompletionForElement(e: Element): string {
+    let autocompletion = `<${e.tagName}`;
+    let tabindex = 1;
+    if (e.attributes.length) {
+      autocompletion += ` $${tabindex++}`;
```

Comment:
Correct, you will need https://github.com/Polymer/atom-plugin/pull/41 to test it.